app/controllers/carts_controller.rb

```diff
@@ -18,6 +18,7 @@ def create
   def show
     @cart_total = grand_total
     @order_items = OrderItem.where(id: session[:cart])
+    session[:target_page] = '/cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.